tests/phpunit/api/v3/ActivityTest.php

```diff
@@ -1285,36 +1272,12 @@ public function testActivityUpdateWithStatus() {
       'activity_name' => 'Test activity type',
     ];
 
-    $result = $this->callAPISuccess('activity', 'create', $params);
-    $this->assertEquals($result['id'], $activity['id']);
```

Comment:
And this one although params are backwards.

Reply:
in this case just the redundant assert is removed